src/main/java/racing/model/car/Result.java

```diff
@@ -0,0 +1,6 @@
+package racing.model.car;
+
+public class Result {
+	public static final boolean WINNER = true;
+	public static final boolean LOSER = false;
+}
```

Comment:
enum 으로 관리하는게 가독성이 더 좋아보여요! 

```suggestion
public enum Result {
	WINNER, LOSER
}
```

이렇게 하면, 추후 요구사항 중 2등도 포함이라면, 여기에서 2등만 추가하면 될것 같아요 👍